README.md

```diff
@@ -21,18 +21,19 @@ transpiled es6 classes rather than `createClass` as of v6.
 
 ## Sections covered in this README
 
-* [Installation](#installation)
```

Comment:
Sorry about this diff from `*` to `-` in the README. It seems like the pre-commit hook is doing it (I guess I don't have the same version of Prettier?). What do?

Reply:
that's an excellent question. Can you tell it to not kick in for .md files in this specific case?

Reply:
To be honest, I don't know how to disable it. Any idea?

Reply:
Nevermind, it looks like the `--no-verify` option works...updating